main.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"cloud.google.com/go/pubsub"
+)
+
+func main() {
+
+	err := publishMessage("pubsub1", "testing1", data{
+		messageID:    "1",
+		requestBody:  "api request body",
+		responseBody: "api response body",
+	})
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+}
+
+type data struct {
+	messageID		string
+	requestBody		string
+	responseBody	string
+}
+
+// PublishMessage publishes messages to already created topics
+func publishMessage(projectID, topicID string, msg data) error {
+	ctx, cancel := context.WithTimeout(context.Background(), 30 * time.Second)
```

Comment:
I would recommend creating a config object, where the timeout in seconds can be one of the items in the config object. This config object should be passed in by the user when initializing the client. 
Also, for a library like ours, it's better to accept the context from the callers of our library, and not using context.Background. 

With context.Background, we lose information about the life cycle of that context.